### ISSUE TITLE
📖 USER STORY: Homepage

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/elevate-hackathon-2024#26]
### User Story

As a **user**, I can **navigate the site homepage** so that I can **get a better understanding of the product offering**.

### Acceptance Criteria

- Homepage has several sections
- Homepage has a jumbotron/hero that helps user understand the site purpose easilt
- Homepage contains text about the company and product(s)

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Create and style the homepage hero
- [x] Add section explaining the product
- [x] Add section explaining the tech behind it
- [x] Add team section
- [x] Ensure page is responsive on different screen sizes

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] Add in animations (Nice to have)
